controllers/discovery/initDiscoveryDomainData.js

```diff
@@ -14,6 +15,7 @@ const InitDiscoveryDomainData = async (req, res) => {
 
   try {
     let suggestedDomainsQuery = `SELECT 
+                C.domain_page_id,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `C.domain_page_id` to the SELECT statement seems unnecessary as it is already being selected as `domain_id_followed`. If there's no specific reason for this, consider removing it to avoid redundancy.
```diff
- C.domain_page_id,
  C.domain_name, C.short_description, C.logo, C.credder_score,
  C.domain_page_id AS domain_id_followed, 
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->